manifests/params.pp

```diff
@@ -87,6 +89,29 @@
       $package_prefix          = 'php-'
       $compiler_packages       = ['gcc', 'gcc-c++', 'make']
       $manage_repos            = false
+      $root_group              = 'root'
+    }
+    'FreeBSD': {
+      $config_root             = '/usr/local/etc'
+      $config_root_ini         = "${config_root}/php"
+      # No common packages, because the required PHP base package will be
+      # pulled in as a dependency. This preserves the ability to choose
+      # any available PHP version by setting the 'package_prefix' parameter.
+      $common_package_names    = []
+      $common_package_suffixes = ['extensions']
+      $cli_inifile             = "${config_root}/php-cli.ini"
+      $dev_package_suffix      = ''
+      $fpm_config_file         = "${config_root}/php-fpm.conf"
+      $fpm_inifile             = "${config_root}/php.ini"
+      $fpm_package_suffix      = ''
+      $fpm_pool_dir            = "${config_root}/php-fpm.d"
+      $fpm_service_name        = 'php-fpm'
+      $fpm_user                = 'www'
+      $fpm_group               = 'www'
+      $package_prefix          = 'php5-'
```

Comment:
Not 100% sure, but ```php56-``` might be a better choice. I had problems on a clean system using pkgng binary packages with the defaults (php5 is 5.4, while the pear package requires 5.6)